frontend/src/store/configurate-episode/actions.ts

```diff
@@ -7,10 +7,13 @@ import {
 } from 'common/types/types';
 import { getFileFromFileList, getDataUrl } from 'helpers/helpers';
 import { ActionType } from './common';
-import { DEFAULT_PODCAST_ID } from 'common/constants/constants';
 import { NotificationMessage, NotificationTitle } from 'common/enums/enums';
 
-const createEpisode = createAsyncThunk<Episode, EpisodeFormPayload, AsyncThunkConfig>(
+type CreateEpisodPayload = EpisodeFormPayload & {
```

Comment:
```suggestion
type CreateActionEpisodPayload = EpisodeFormPayload & {
```
and move it to the `frontend/src/common/types/episode`